Modrinth/overrides/config

```diff
@@ -0,0 +1 @@
+../../Packwiz/config
```

Comment:
For the sake of keeping the original folder structure, please also create empty folders with an empty file `.gitkeep` on it.

Reply:
Which folders? If `Modrinth/overrides/mods/.gitkeep` is the only one needed, done in d7a2632.

Reply:
I didn't read the wiki throughly enough so I thought there might be some more folder structure, as in MultiMC. But yeah, `mods` seems to be the only one.